### PR TITLE
Refactoring, translations, etc.

### PR DESCRIPTION
**Pull request by N-Wouda**
- [x] Adding some more notifications/feedback on events
- [x] Factoring out duplicate code (some stuff exists in several locations, not needed)
- [x] Translations. Non-exhaustive.
- [x] Comments/JSDocs.

### CODE REVIEW

File: src/controllers/levelcontroller.js [hunk 1]
```diff
@@ -2,18 +2,13 @@ class LevelController extends Controller
...
 
-        // TODO Make this prettier, gets bloated when adding more types of levels
-        if (currentLevel.type == 'money') {
-            completed = this.checkGoalMoney(currentLevel);
-        } else if (currentLevel.type == 'satisfaction') {
-            completed = this.checkGoalSatisfaction(currentLevel);
-        }
-
-        if (completed) {
+        if ((cases[currentLevel.type].bind(this))(currentLevel)) {
```
Comment (N-Wouda) [inline src/controllers/levelcontroller.js:11]: @lukko this is what I had in mind here :)

Comment (lukko) [reply src/controllers/levelcontroller.js:11]: Looks much cleaner!

File: src/controllers/initgamecontroller.js [hunk 2]
```diff
@@ -31,7 +31,10 @@ class InitGameController extends Controller
         );
 
         $(document).ready(function () {
-            HelpController.startIntroTour();
+            if (!Cookies.get('hasVisited')) {
+                HelpController.startNavbarTour(true);
+                Cookies.set('hasVisited', true, {expires: 7});
+            }
```
Comment (lukko) [inline src/controllers/initgamecontroller.js:38]: This wasn't working for me.
Might have been some error from my side.
If it does work, then this is of course better :)

**Comment by N-Wouda** [comment]
I'm not getting any errors, so I guess it works?

File: src/controllers/initgamecontroller.js (hunk 2 above)
Comment (N-Wouda) [reply src/controllers/initgamecontroller.js:38]: I'm not getting any errors, so I guess it works?

Comment (lukko) [reply src/controllers/initgamecontroller.js:38]: Then it was indeed an error on my side :D

Comment (N-Wouda) [reply src/controllers/initgamecontroller.js:38]: I'll add a comment that we need to check this and leave it like this for now.

**Pull request merged**